lib/iris/fileformats/netcdf/loader.py

```diff
@@ -719,18 +719,19 @@ def set(
             Kwargs specifying chunksizes for dimensions of file variables.
             Each key-value pair defines a chunksize for a named file
             dimension, e.g. {'time': 10, 'model_levels':1}.
+            Values of -1 will lock the chunk to the size of it's shape.
 
         Notes
         -----
         This function acts as a contextmanager, for use in a 'with' block.
 
-        Example:
-
-        #todo
-        # >>> from iris.fileformats.netcdf.loader import CHUNK_CONTROL
-        # >>> from iris import sample_data_path
-        # >>> with CHUNK_CONTROL.set('var1', model_level=1, time=50):
-        # ...     cubes = iris.load(sample_data_path("toa_brightness_stereographic.nc"))
+        >>> import iris
+        >>> from iris.fileformats.netcdf.loader import CHUNK_CONTROL
+        >>> with CHUNK_CONTROL.set("air_temperature", time=180, latitude=-1):
+        ...     cube = iris.load(iris.sample_data_path("E1_north_america.nc"))[0]
+        >>>
+        >>> print(cube.lazy_data().chunksize)
```

Comment:
```suggestion
        >>> print(cube.core_data().chunksize)
```

Using `lazy_data()` confuses the message somewhat, because if the `Cube` has been realised, `lazy_data()` will still return a Dask version of the data, but it will NOT have the chunks it had when it was originally loaded. `core_data()` will either have the correct `chunksize`, or return an `AttributeError` if is a NumPy array - it will never give the wrong answer. Sorry I missed that when we were writing the tests, but careful messaging is more important in the documentation.

Some more to change in the `.rst` file too.

### [`.lazy_data()`](https://scitools-iris.readthedocs.io/en/stable/generated/api/iris.cube.html#iris.cube.Cube.lazy_data)

>...
>
>If the Cube data have already been loaded (for example by calling [data()](https://scitools-iris.readthedocs.io/en/stable/generated/api/iris.cube.html#iris.cube.Cube.data)), the returned Array will be a view of the loaded cube data represented as a lazy array object. Note that this does _not_ make the Cube data lazy again; the Cube data remains loaded in memory.

### [`core_data()`](https://scitools-iris.readthedocs.io/en/stable/generated/api/iris.cube.html#iris.cube.Cube.core_data)

>...
>
>If this [Cube](https://scitools-iris.readthedocs.io/en/stable/generated/api/iris.cube.html#iris.cube.Cube) has lazy data, accessing its data array via this method will not realise the data array. This means you can perform operations using this method that work equivalently on real or lazy data, and will maintain lazy data if present.